_emails/kalamazooMi.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kalamazoo, MI
+permalink: "/kalamazooMi"
+name: Letter to City and State Representatives
+state: MI
+city: Kalamazoo
+layout: email
+recipients:
+- mayor@kalamazoocity.org
+- knotte@kalamazoocity.org
+- urbanj@kalamazoocity.org
+- hessj@kalamazoocity.org
+- praedelc@kalamazoocity.org
+- cokcitymanager@kalamazoocity.org
+- Stephanie.Moore@kalcounty.com
+- Zachary.Bauer@Kalcounty.com
+- Tracy.Hall@kalcounty.com
+- Michael.Seals@kalcounty.com
+- Julie.Rogers@kalcounty.com
+- Jennifer.Aniano@kalcounty.com
+- Roger.Tuinier@kalcounty.com
+- John.Gisler@kalcounty.com
+- Christine.Morse@kalcounty.com
+- Mike.Quinn@kalcounty.com
+- Meredith.Place@kalcounty.com
+- SenSMcCann@senate.michigan.gov
+- jonhoadley@house.mi.gov
+subject: Regarding the funding of KDPS
+body: "Hello, \n\nI am a constituent living in Kalamazoo County and I am writing to you 
+regarding recent police excessive force at the protests against police brutality 
+as well as Kalamazoo Department of Public Safety�s place in our budgets.
+The police�s behavior at the peaceful protests in early June were been aggressive, 
+unnecessary, and intentionally provocative. It�s clear from all footage that the only 
+people in danger were the protesters, surprised by police cruisers speeding towards them, 
+the National Guard boxing them in, and tear gas, a tactic not allowed in war but evidently 
+perfectly fine for a medium size city�s oversized police. \n\nThis kind of behavior is a 
+long standing pattern with KDPS, a police force with multiple questionable in custody deaths, 
+excessive force accusations, a documented habit of pulling over predominantly black drivers 
+and enough unnecessary lethal equipment to outfit a small army. The time for hiding behind 
+canned apologies, promises of diversity training and �community policing� is over. It�s time 
+to reduce police spending and reallocate those funds to programs that have real tangible benefits, 
+especially to communities harmed by decades of over-policing and discrimination. \n\nKalamazoo Public 
+School officials have been sounding the alarm that due to COVID-19 tax shortfalls, they could be
+facing unprecedented cuts to per pupil funding. Yet KDPS is accepting applications for new 
+officers despite already having more employees than any other city department. Public health 
+workers in Kalamazoo have had to sew their own face masks to serve consumers safely during the 
+pandemic. Yet KDPS has a stockpile of face shields they are using for public intimidation, not 
+protection. Kalamazoo Public Safety takes up 48% of the entire general fund budget for 2020. Other 
+public services account for a paltry 8% and parks and recreation a depressing 4% of budget. 
+That is outrageous and needs to change. Kalamazoo needs dollars for services that actually 
+improve and lift up our community. Dollars for education, for food assistance, for housing, 
+for people. Not policing.\n\nSincerely,\n[your name]\n"
```

Comment:
Let's include name, email, phone number, address

apparently emails without this are being filtered out.

Reply:
Understood, the edits will be in momentarily